pkg/controller/v1alpha2/applicationconfiguration/applicationconfiguration.go

```diff
@@ -67,16 +68,17 @@ const (
 )
 
 // Setup adds a controller that reconciles ApplicationConfigurations.
-func Setup(mgr ctrl.Manager, l logging.Logger) error {
+func Setup(mgr ctrl.Manager, args *controller.Args, l logging.Logger) error {
```

Comment:
I think it will be more convenient if we directly use struct instead of pointer and won't have risk to get panic.

Because we will use `args.DefaultRevisionLimit` below.

Reply:
👌